src/index.ts

```diff
@@ -0,0 +1,65 @@
+import { Editor } from 'grapesjs'
+import { KeymapsDialogManager } from './KeymapsDialogManager'
+
+const cmdKeymapsDialog = 'open-keymaps-dialog'
+
+export default (editor: Editor, opts = {}): void => {
+  const options = {
+    ...{
+      longPressKey: 'shift',
+      longPressDuration: 800,
+      shortcut: 'shift+k',
```

Comment:
i understand "shift+?" doesnt work, maybe just "?" ?

Reply:
yes right, i understand now

shift+k for me is search or "go to"

let's discuss this in the forums, it's ok for now with shift+k

Reply:
especially because it can be changed with options